content/en/project/getInvolved/_index.md

```diff
@@ -28,40 +28,44 @@ Bug reports, feature requests, documentation problems, and questions are posted
 
 * [Medley](https://github.com/Interlisp/medley): Lisp code, build scripts and GitHub automation
 * [maiko](https://github.com/Interlisp/maiko): The C-coded virtual machine
-* [this web site](https://github.com/Interlisp/Interlisp.github.io#README)
+* [the Interlisp.github.io repository](https://github.com/Interlisp/Interlisp.github.io#README) of the interlisp.org web site
```

Comment:
How about keeping it consistent with the Medley and Maiko sites (should Maiko be capitalized?)
[Interlistp.github.io](xxx): The Hugo source code for the interlisp.org site